packages/run-protocol/src/reserve/doc.md

```diff
@@ -0,0 +1,129 @@
+# Reserve 
+
+### addIssuer
+
+```mermaid
+sequenceDiagram
+
+participant caller
+participant addIssuer
```

Comment:
`addIssuer` is [a method of the reserve's creator facet](https://github.com/Agoric/agoric-sdk/blob/2b5e79c8671d9f3cd891e28b0b20de8df4e3bca9/packages/run-protocol/src/reserve/assetReserve.js#L167).
```suggestion
participant reserveCreatorFacet
```